Makefile

```diff
@@ -89,3 +90,14 @@ uninstall: scripts_uninstall modules_uninstall
 lunatik_sym.h: $(LUA_API)
 	${shell ./gensymbols.sh $(LUA_API) > lunatik_sym.h}
 
+moontastik_install_%:
+	[ $* ] || (echo "Which target would you like to install?" && exit 1)
```

Comment:
IMHO, a better idea is to print something like `usage: make moontastik_install <target>`